EIPS/bigint_modexp.md

```diff
@@ -0,0 +1,75 @@
+### Parameters
+
+* `GQUADDIVISOR: 20`
+
+### Specification
+
+At address 0x00......05, add a precompile that expects input in the following format:
+
+    <length_of_BASE> <length_of_EXPONENT> <length_of_MODULUS> <BASE> <EXPONENT> <MODULUS>
+    
+Where every length is a 32-byte left-padded integer representing the number of bytes to be taken up by the next value. Call data is assumed to be infinitely right-padded with zero bytes, and excess data is ignored. Consumes `floor(max(length_of_MODULUS, length_of_BASE) ** 2 * max(length_of_EXPONENT, 1) / GQUADDIVISOR)` gas, and if there is enough gas, returns an output `(BASE**EXPONENT) % MODULUS` as a byte array with the same length as the modulus.
```

Comment:
Why? If the length of the modulus, base and the exponent are 256 bytes, then max(256, 256)**2 * max(256, 1) / 20 = 256**2 * 256 / 20 = 838860. Don't see the problem.